app/components/Ability.tsx

```diff
@@ -58,7 +61,7 @@ export function Ability({
         setIsDragTarget(false);
         onDrop?.(event);
       }}
-      type="button"
```

Comment:
https://user-images.githubusercontent.com/38327916/193895399-51aa9b88-9fa4-40aa-995b-c22a6006cfc2.mov

Is it important to remove this attribute? Maybe only conditionally reading readonly? See video for the bug it will cause when adding new build (tries to submit new build on ability click)

Reply:
Sorry, I inadvertently removed `type="button"`. Should absolutely stay in. I'll push a fix. Thanks!

Reply:
Fixed in https://github.com/Sendouc/sendou.ink/pull/990/commits/36deb0c58820edf464cb35f352c911ee402e6f27